packages/mui-base/src/MenuButton/MenuButton.test.tsx

```diff
@@ -155,7 +151,9 @@ describe('<MenuButton />', () => {
             button.focus();
           });
 
-          await userEvent.keyboard(`{${key}}`);
+          const user = userEvent.setup();
```

Comment:
Do you know if this is required in each test and shouldn't be called just once at the top?

Reply:
I initially thought about this but I believed tests should be independent. However, upon your inquiry, I researched further and discovered relevant documentation: https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent. Additionally, I found a helpful explanation on Stack Overflow: https://stackoverflow.com/a/76457218/8146534. 

Moreover, I came across this insightful discussion: https://github.com/testing-library/user-event/discussions/1052#discussioncomment-3641692. 

Furthermore, I followed the recommendation to invoke `setup()` before rendering the component.

Apologies for the numerous commits; I encountered some issues with pnpm v8.

New `browserstack-force` CI run for Safari: https://app.circleci.com/pipelines/github/mui/base-ui/1345/workflows/d42c306c-32ef-4237-af04-90d77a371586/jobs/7691

Reply:
Fair enough, thank you!